app/routes/my-projects/archive.js

```diff
@@ -1,4 +1,15 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class MyProjectsArchiveRoute extends Route {
+  @service
+  currentUser;
+
+  @service
+  store;
+
+  async model() {
+    // Use this endpoint for now. This will need to be updated when the backend is finalized.
+    return this.store.query('project', { project_lup_status: 'archive' });
```

Comment:
According to the LUP Tab wireframes it looks like we'll need some milestones on the right-hand-side of the proejct cards, so we'll probably also need to "sideload" milestones with the projects. We can do so by adding  the `included` property to the project query:
```
{ project_lup_status: 'upcoming', include: 'milestones' }
```